design/open/spark-outputcommitter/lakeFS-output-committer-execution-plan.md

```diff
@@ -0,0 +1,38 @@
+# LakeFS Output Committer - Execution Plan
+[LakeFS Output Committer Proposal](https://github.com/treeverse/lakeFS/blob/master/design/open/spark-outputcommitter/committer.md)
+
+## Overview
+Implementing lakeFS Output Committer meaning implementing the class [FileOutputCommitter](https://hadoop.apache.org/docs/stable/api/org/apache/hadoop/mapreduce/lib/output/FileOutputCommitter.html) and using lakeFS client to create/merge branches as part of setup and commit job/tasks (this is the [sample flow](https://github.com/treeverse/lakeFS/blob/master/design/open/spark-outputcommitter/committer.md#sample-flow))
+
+### Milestone 1
+The first milestone is focused on writing a file with single mode (probably text) and one file format (probably append).
+At this stage we'll implement `commit`, `setup`, `abort` class methods.
+
+### Milestone 2
```

Comment:
It looks like milestone 2 is pretty packed. Wouldn't it be sufficient to complete support for all file formats in this milestone, and support all modes in another? 
(If we consider a milestone as a release point, it would make our dev-cycle more agile)

Reply:
It was updated and I had a mistake and didn't update here correctly. Will now push the relevant changes :)